source/eve_evaluate.cpp

```diff
@@ -325,9 +326,9 @@ void apply_layout_parameters(layout_attributes_t& data, const dictionary_t& para
 
                 layout_attributes_t::spacing_t::iterator dest_iter(data.spacing_m.begin() + 1);
 
-                for (array_t::const_iterator iter(spacing_array.begin());
-                     iter != spacing_array.end(); ++iter) {
-                    *dest_iter = iter->cast<int>();
+                for (array_t::const_iterator spacing_iter(spacing_array.begin());
```

Comment:
I think a single character name would actually be clearer in this case.  Also, though, no raw loops would alleviate the need to name these trivial things everywhere.

Reply:
Replaced this one with `transform()`